packages/frontend/package.json

```diff
@@ -50,7 +50,7 @@
     "karma-jasmine": "~3.0.1",
     "karma-jasmine-html-reporter": "^1.4.2",
     "protractor": "~7.0.0",
-    "puppeteer": "^3.3.0",
+    "puppeteer": "latest",
```

Comment:
this is a bit weird, can we revert this dependency update?  If we do want to upgrade puppeteer, we can do:

```
npm i puppeteer@latest --save-dev
```

But this should be in another PR>.